mcs/class/System.Windows.Forms/System.Windows.Forms.RTF/CharsetToCodepage.cs

```diff
@@ -0,0 +1,78 @@
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+// Copyright (c) 2020 Karl Scowen
+//
+// Authors:
+//  Karl Scowen		<contact@scowencomputers.co.nz>
+//
+//
+
+
+namespace System.Windows.Forms.RTF {
+	internal static class CharsetToCodepage {
+		public static int Translate(CharsetType charset)
+		{
+			switch (charset) {
+				case CharsetType.ANSI:
```

Comment:
Technically, this should be `Encoding.Default.WindowsCodePage` which is system dependent. I don't know how often is that used in practice. The same applies to OEM but that's obscure one and I don't think it has equivalent in managed world.

Reply:
According to the specs (https://web.archive.org/web/20190708132914/http://www.kleinlercher.at/tools/Windows_Protocols/Word2007RTFSpec9.pdf, page 19), ANSI (\fcharset0) means code page 1252 and OEM means 850, but you're absolutely right about "General" (more correctly "Default"). Will change that.

Reply:
I checked the specification. The table is there only for convenience but now I see it matched GDI charset numbers which are used in various other specifications (eg. WMF). So, yeah, "General" should be `Encoding.Default` and ANSI can stay as-is.